src/MixedModels.jl

```diff
@@ -15,23 +15,21 @@ using ProgressMeter
 using SparseArrays
 using StaticArrays
 using Statistics
+using StatsAPI
 using StatsBase
 using StatsModels
 using StructTypes
 using Tables
 
 using LinearAlgebra: BlasFloat, BlasReal, HermOrSym, PosDefException, copytri!
 using Base: Ryu, require_one_based_indexing
-using GLM: Link, canonicallink, linkfun, linkinv
+using DataAPI: levels, refpool, refarray, refvalue
+using GLM: Link, canonicallink, linkfun, linkinv, dispersion, dispersion_parameter
+using NLopt: Opt
 using StatsModels: TableRegressionModel
-
 using StatsFuns: log2π, normccdf
 
-import Base: *
-import DataAPI: levels, refpool, refarray, refvalue
-import GLM: dispersion, dispersion_parameter
-import NLopt: Opt
-import StatsBase: fit, fit!
+import Base: ==
```

Comment:
Why is the import of `==` required?  I couldn't find a case of an `==` method being defined.

Reply:
I think that line is redundant.  I was able to comment it out and still pass the tests.

Reply:
oops, I missed removing it.

Reply:
```suggestion
```